content/applications/hr/recruitment/refuse_applicant.rst

```diff
@@ -0,0 +1,163 @@
+       | :guilabel:`Duplicate`
+       | :guilabel:`Spam`
+   * - :guilabel:`Recruitment: Not interested anymore`
+     - | :guilabel:`Refused by Applicant: don't like job`
+       | :guilabel:`Refused by Applicant: better offer`
+       | :guilabel:`Refused by Applicant: salary`
+
+Additional refusal reasons :ref:`can be created and existing ones can be modified or deleted
+<recruitment/new-refuse>`.
+
+After selecting a refuse reason, click the :guilabel:`Refuse` button to refuse the applicant. An
+:ref:`email is sent <recruitment/email>` to the applicant, and the applicant card is no longer
+visible in the job position Kanban view.
+
+.. image:: refuse_applicant/refuse.png
+   :align: center
+   :alt: An applicant's card with the refused banner appearing in the top-right corner in red.
+
+.. note::
+   Pre-configured email templates in Odoo sometimes use dynamic placeholders, which are customized
+   pieces of data that populate dynamic content. For example, if the applicant's name is a piece of
+   dynamic content, the applicant's name appears anytime that dynamic placeholder appears on the
+   email template. For more detailed information on email templates, refer to the
+   :doc:`../../general/companies/email_template` documentation.
+
+.. _recruitment/new-refuse:
+
+Create or modify refuse reasons
+-------------------------------
+
+Refuse reasons allow recruiters to document why an applicant was not a good fit, and send specific
+refusal reason email templates to the applicant.
+
+To view and configure refuse reasons, navigate to :menuselection:`Recruitment app --> Configuration
+--> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.
+
+To create a new refuse reason, click the :guilabel:`New` button in the top-left corner. A blank
+line appears at the bottom of the :guilabel:`Description` column.
+
+Type in the new refuse reason in the field. It is recommended to enter a reason that is short and
+concise, such as `offer expired` or `withdrew application`.
+
+Then, in the :guilabel:`Email Template` field, click on the field to reveal a drop-down menu.
+Select an :guilabel:`Email Template` from the list to be used when this refuse reason is selected.
+
+If a new :guilabel:`Email Template` is desired, type in the name for the new template in the field.
+Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
+window appears.
+
+Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.
+Enter the email content in the :guilabel:`Content` tab. Make any other desired modifications to the
+template, then click :guilabel:`Save & Close` to save the template and return to the
+:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
+Template` field.
+
+To make modifications to an existing refuse reason, click on the line, and make any desired changes.
+
+.. _recruitment/email:
+
+Send refusal email
```

Comment:
I thought about that- but I did adda sentence after the seciton you mentioned, and there is a link in the text to jump to the send email section. So because of that, I think it's ok- they can go in order (just with a click).